server/game/cards/events/01/winteriscoming.js

```diff
@@ -0,0 +1,18 @@
+const ChallengeEvent = require('../../challengeevent.js');
+
+class WinterIsComing extends ChallengeEvent {
+
+    // TODO implement restriction "(Max 1 per challenge.)"
```

Comment:
This probably means bringing events under the effects engine somehow, @ystros FYI